packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../widgets.dart';
+import '../../widgets/localized_view.dart';
+import 'updates_other_software_model.dart';
+
+const _kButtonPadding = EdgeInsets.only(right: 20, bottom: 20);
+
+class UpdatesOtherSoftwarePage extends StatefulWidget {
+  @override
+  _UpdatesOtherSoftwarePageState createState() =>
+      _UpdatesOtherSoftwarePageState();
+
+  static Widget create(BuildContext context) {
+    return ChangeNotifierProvider(
+      create: (context) =>
+          UpdateOtherSoftwareModel(installationMode: InstallationMode.normal),
+      child: UpdatesOtherSoftwarePage(),
+    );
+  }
+}
+
+class _UpdatesOtherSoftwarePageState extends State<UpdatesOtherSoftwarePage> {
+  @override
+  Widget build(BuildContext context) {
+    final model = context.watch<UpdateOtherSoftwareModel>();
+    return LocalizedView(
+      builder: (context, lang) => Scaffold(
+        appBar: AppBar(
+          title: Text(lang.updatesOtherSoftwarePageTitle),
+          automaticallyImplyLeading: false,
+        ),
+        body: Container(
+          padding: const EdgeInsets.all(20),
```

Comment:
could the padding be around the description label and as content padding of the radio list tiles instead?

![image](https://user-images.githubusercontent.com/140617/118011395-45848380-b350-11eb-8179-176e2c7534f1.png)

Reply:
done